Lesson02_Task02.py

```diff
@@ -0,0 +1,53 @@
+"""
```

Comment:
For the next time please do not mix all files within pull request. Better to add one certain hometask as separate PR. 
Ex. HomeTask2 -> pull request #2, HomeTask3 -> pull request #3  ...